src/bin/imageprimaries.rs

```diff
@@ -36,28 +37,27 @@ use petalo::Length;
 use geometry::units::{mm, mm_};
 use petalo::fov::FOV;
 use petalo::image::Image;
-use petalo::io::hdf5::{read_table, Primary};
+use petalo::io::mcreaders::{read_primaries, MCPrimary};
 type L = Lengthf32;
```

Comment:
Of course, no problem.

Many of my names start off as `xxx` and `yyy`. Then, hacking on the implementation and use, I gradually get a better feeling for what the thing should be called, and the name evolves. Good names are vitally important but also costly to get right, so it's perfectly ok to postpone coming up with a good name so that you can concentrate on getting the implementation done. As long as we remember that the names should be improved.